### PR TITLE
Apps acquisitions API

### PR DESCRIPTION
**Pull request by Aracho1**
- We have created a new API for sending new app subscriptions to BigQuery directly using a lambda instead of sending via Ophan
- Instructions on how to test locally can be found on README

### CODE REVIEW

File: support-lambdas/acquisition-events-api/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Acquisition Events Api
+- API for sending in-app acquisitions(subscriptions) to BigQuery using a lambda
+
+## Testing
+To test locally:
+- Uncomment <code>@Ignore</code> from <code>LambdaSpec.scala</code>
```
Comment (tomrf1) [inline support-lambdas/acquisition-events-api/README.md:6]: "Comment out..."

Comment (tomrf1) [reply support-lambdas/acquisition-events-api/README.md:6]: also, you can use ` instead of <code> tags

Comment (Aracho1) [reply support-lambdas/acquisition-events-api/README.md:6]: 👍

**Review by tomrf1 — APPROVED** [review]
👍 great stuff!
let's add alarms in a follow up PR

**Comment by Aracho1** [comment]
thanks!

**Pull request merged**